### ISSUE TITLE
Thunderbird won't open (GLib-GIO-ERROR) (Bugzilla Bug 2375)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#2375]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 2375
Date: 2018-10-03 14:46:40 +0000
From: @benbullard79
To:   @berolinux
CC:   @benbullard79, bugs@openmandriva.org, @itchka, nix.or.die@gmail.com, nobodydead@gmail.com, om-security@ml.openmandriva.org, @tpgxyz

Last updated: 2018-10-04 05:35:16 +0000

## Comment 18653
Date: 2018-10-03 14:46:40 +0000
From: @benbullard79

As the title says Thunderbird won't open. From Konsole output we see these errors:

(thunderbird:8701): Gtk-WARNING **: 07:42:50.646: Theme parsing error: gtk-contained.css:3406:17: Using Pango syntax for the font: style property is deprecated; please use CSS syntax

(thunderbird:8701): GLib-GIO-ERROR **: 07:42:51.208: No GSettings schemas are installed on the system
Trace/breakpoint trap (core dumped)

I think it is the GLib-GIO-ERROR that is likely the issue here.

I'm in Cooker at the moment and if I try to add an attachment to this bug report firefox crashes. Yeah. Another bug.

Anyway I'll try to get some more info as I have time.

## Comment 18654
Date: 2018-10-03 14:48:54 +0000
From: @benbullard79

Created attachment 1653
strace thunderbird

> Attached file: strace_thunderbird.txt (text/plain, 933754 bytes)
> Description:   strace thunderbird

## Comment 18655
Date: 2018-10-03 14:52:09 +0000
From: @benbullard79

Created attachment 1654
omv bug report log

> Attached file: omv-bug-report.log.xz (application/x-xz, 79412 bytes)
> Description:   omv bug report log

## Comment 18656
Date: 2018-10-03 14:54:25 +0000
From: @benbullard79

Created attachment 1655
Konsole output for thunderbird

> Attached file: thunderbird-cli.txt (text/plain, 7773 bytes)
> Description:   Konsole output for thunderbird

## Comment 18657
Date: 2018-10-03 15:07:04 +0000
From: @benbullard79

Created attachment 1656
sudo strace -d thunderbird

If I'm not doing what is needed with strace let me know what I need to do.

> Attached file: strace_-d_thunderbird.txt (text/plain, 162529 bytes)
> Description:   sudo strace -d thunderbird

## Comment 18658
Date: 2018-10-04 05:33:56 +0000
From: @benbullard79

Should have mentioned that these packages were not installed on my system by default but I did install them before filing this report:

$ rpm -qa | grep gsettings
lib64gsettings-desktop-schemas-gir3.0-3.28.1-1.x86_64
gsettings-desktop-schemas-3.28.1-1.x86_64

Not related but noteworthy is that I found the os-prober was not installed on this system. That has to be an error. I installed the same ISO on another computer and os-prober was installed so I don't know what is going on. Also on the other computer Thunderbird works.

Also I lost sight of the fact that the mass rebuild is not yet finished, or maybe I just in the excitement of an actual ISO that actually will install something did not think about it.

Thus I believe I should for now, at least, mark this as RESOLVED/INVALID. After the mass rebuild is finished and a new ISO made based on that then I will revisit this and a number of other issues and if warranted will re-open this bug.

## Comment 18659
Date: 2018-10-04 05:35:16 +0000
From: @benbullard79

In other words I'm saying I jumped the gun on filing this bug report. We should wait until mass rebuild is done and an ISO made based on that Cooker and then start chasing bugs.
